fetch_data.py

```diff
@@ -200,17 +227,21 @@ def deserialize_order_items(region, redo_urls, func):
     return deserialized_results, redo_urls
 
 
+def parse_history_results(results, histories):
+    for result in results:
+        result_item_id = int(result.url.split("=")[-1])
+        item_history = json.loads(result.text)
+        histories[result_item_id] = item_history
+
+
 def deserialize_history_chunk(history_urls, histories):
     for history_chunk in history_urls:
         results, redo_urls, error_timer = futures_results(create_history_futures(history_chunk))
-        for result in results:
-            result_item_id = int(result.url.split("=")[-1])
-            item = json.loads(result.text)
-            histories[result_item_id] = item
+        parse_history_results(results, histories)
         pause_futures(error_timer, f"Sleeping history fetching due to error timer being {error_timer} seconds")
         while len(redo_urls) != 0:
             addtl_results, redo_urls, error_timer = futures_results(create_history_futures(redo_urls))
```

Comment:
https://github.com/Xavier-J-Ortiz/market_hub_puller/pull/29/files#diff-afcd4836ccef17f66d84e9b8c2b38ed8fd9566a9e184bccc8e39dd759e402f2eR237

Same question applies here.

Reply:
#32 addresses this. Resolving this particular comment.